[INFO -- 2018-09-15 12:11:36]
Starting flip_value script
version: 1.1
[INFO -- 2018-09-15 12:11:36]
Initialization strings:
[INFO -- 2018-09-15 12:11:36]
(1) file /tmp/lreg/lreg
[INFO -- 2018-09-15 12:11:36]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-09-15 12:11:36]
initSignal:0
[INFO -- 2018-09-15 12:11:36]
endSignal:2
[INFO -- 2018-09-15 12:11:36]
faultModel:2
[EXCEPTION -- 2018-09-15 12:11:38]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-09-15 12:11:38]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[EXCEPTION -- 2018-09-15 12:11:38]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-09-15 12:11:39]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec1801 in __GI_abort () at abort.c:79
#2  0x00007ffff6f0a897 in __libc_message (action=action@entry=do_abort, fmt=fmt@entry=0x7ffff7037b9a "%s\n") at ../sysdeps/posix/libc_fatal.c:181
#3  0x00007ffff6f1190a in malloc_printerr (str=str@entry=0x7ffff7035d88 "free(): invalid pointer") at malloc.c:5350
#4  0x00007ffff6f18e1c in _int_free (have_lock=0, p=0x555555771544, av=0x7ffff726cc40 <main_arena>) at malloc.c:4157
#5  __GI___libc_free (mem=0x555555771554) at malloc.c:3124
#6  0x000055555555772a in __gnu_cxx::new_allocator<double>::deallocate (this=0x55555575a140 <x>, __p=0x555555771554) at /usr/include/c++/7/ext/new_allocator.h:125
#7  0x00005555555571f0 in std::allocator_traits<std::allocator<double> >::deallocate (__a=..., __p=0x555555771554, __n=29) at /usr/include/c++/7/bits/alloc_traits.h:462
#8  0x0000555555556944 in std::_Vector_base<double, std::allocator<double> >::_M_deallocate (this=0x55555575a140 <x>, __p=0x555555771554, __n=29) at /usr/include/c++/7/bits/stl_vector.h:180
#9  0x0000555555556dc5 in std::vector<double, std::allocator<double> >::_M_realloc_insert<double const&> (this=0x55555575a140 <x>, __position=0, __args#0=@0x7fffffffd958: -26) at /usr/include/c++/7/bits/vector.tcc:448
#10 0x000055555555655e in std::vector<double, std::allocator<double> >::push_back (this=0x55555575a140 <x>, __x=@0x7fffffffd958: -26) at /usr/include/c++/7/bits/stl_vector.h:948
#11 0x0000555555555b9f in main (argc=3, argv=0x7fffffffde68) at lreg.cpp:175

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping a bit of the pointer
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x7ffff726e8f0 
address max offset: 8
Memory content after  bitflip:0x5b0x480x240xa70xd50x7e0x0f0x32
frame name: __GI___libc_free
symbol name: __malloc_initialize_hook
symbol filename: malloc.c
symbol line: 1859
value: 0x0
value address: 0x7ffff726e8f0 <__malloc_initialize_hook>
Type: The type is a pointer.
Type sizeof: 8

[INFO -- 2018-09-15 12:11:39]
Fault Injection Successful after 3.6521687507629395s
[INFO -- 2018-09-15 12:11:39]
event type: exit
[EXCEPTION -- 2018-09-15 12:11:39]
exit code: no exit code available
